src/components/SlateEditor/plugins/mathml/MathML.jsx

```diff
@@ -1,66 +0,0 @@
-/**
- * Copyright (c) 2018-present, NDLA.
- *
- * This source code is licensed under the GPLv3 license found in the
- * LICENSE file in the root directory of this source tree.
- *
- */
-
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-
-class MathML extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { reRender: false };
-  }
-
-  async componentDidUpdate(prevProps) {
-    const { model } = this.props;
-    const { innerHTML } = model;
-    if (prevProps.model.innerHTML !== innerHTML) {
-      // Note: a small delay before a 're-render" is required in order to
-      // get the MathJax script to render correctly after editing the MathML
-      this.setState({ reRender: true });
-      await setTimeout(() => {
-        this.setState({ reRender: false });
-      }, 10);
-    }
-  }
-
-  render() {
-    const { reRender } = this.state;
-    if (reRender) {
-      return null;
-    }
-    const { node, model, attributes } = this.props;
-    return (
-      <span data-cy="math" contentEditable={false} {...attributes}>
-        {node.children.map(text => (
```

Comment:
Går det greit å fjerne denne loopen? Jeg vet ikke om vi hadde det før, men som det er nå så finner jeg ingen eksempler på at node.children har mer enn et element. Vi bruker heller ikke verdiene i arrayen bortsett text.key (som ikke finnes lengre) så jeg ser ikke helt poenget med den

Reply:
Dette er kun for å vise fram matte i en artikkel, sant? Det fungerer uten denne loopen så for meg ser det trygt ut å fjerne den.

Reply:
Bare å fjerne denne så vidt jeg kan se

Reply:
For å vise fram i artikkel, ja